### PR TITLE
Fix member ordering for creating member based channels

### PR DESCRIPTION
**Pull request by ferhatelmas**
since it's sorted now.

### CODE REVIEW

**Comment by ferhatelmas** [comment]
Btw, it is a unit test than an integration test, we could remove it too since covered.

File: test/integration/channels.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ describe('Channels - Constructor', function () {
 
 	it('short version with options', function (done) {
 		const channel = client.channel('messaging', { members: ['tommaso', 'thierry'] });
-		expect(channel.data).to.eql({ members: ['tommaso', 'thierry'] });
+		expect(channel.data).to.eql({ members: ['thierry', 'tommaso'] });
```
Comment (mahboubii) [inline test/integration/channels.js:126]: this test seems to be a better fit for unit tests. is this due to recent changes? the member order should remain the same

Comment (ferhatelmas) [reply test/integration/channels.js:126]: yes, it comes from member based channel creation fix.

I agree with unit.

**Review by vishalnarkhede — APPROVED** [review]

File: test/integration/channels.js (hunk 1 above)
Comment (mahboubii) [reply test/integration/channels.js:126]: oh I see, the sort is applied in place, I think it's better to revert that change

Comment (ferhatelmas) [reply test/integration/channels.js:126]: I will make a fix to sort the copy and move this test to unit tests.

Comment (mahboubii) [reply test/integration/channels.js:126]: @ferhatelmas https://github.com/GetStream/stream-chat-js/pull/592

Comment (ferhatelmas) [reply test/integration/channels.js:126]: ahh, updated this one

**Review by mahboubii — APPROVED** [review]
🌮

**Pull request merged**